e2e-tests/cypress/support/insurance/check-your-business-summary-list.js

```diff
@@ -0,0 +1,181 @@
+
+    assertRow(fieldId, expectedKey, expectedValue);
+  },
+  [COMPANY_ADDRESS]: () => {
+    const fieldId = COMPANY_ADDRESS;
+    const expectedKey = FIELDS[fieldId].SUMMARY.TITLE;
+
+    const { summaryList } = checkYourAnswersPage;
+    const row = summaryList[fieldId];
+
+    cy.checkText(
+      row.key(),
+      expectedKey,
+    );
+
+    // as html, cannot use checkText so checking contains following fields
+    row.value().contains(application.EXPORTER_COMPANY[fieldId].addressLine1);
+    row.value().contains(application.EXPORTER_COMPANY[fieldId].addressLine2);
+    row.value().contains(application.EXPORTER_COMPANY[fieldId].locality);
+    row.value().contains(application.EXPORTER_COMPANY[fieldId].region);
+    row.value().contains(application.EXPORTER_COMPANY[fieldId].postalCode);
+
+    row.changeLink().should('not.exist');
+  },
+  [COMPANY_INCORPORATED]: () => {
+    const fieldId = COMPANY_INCORPORATED;
+    const expectedKey = FIELDS[fieldId].SUMMARY.TITLE;
+
+    const expectedValue = application.EXPORTER_COMPANY[fieldId];
+
+    assertRow(fieldId, expectedKey, expectedValue);
+  },
+  [COMPANY_SIC]: () => {
+    const fieldId = COMPANY_SIC;
+    const expectedKey = FIELDS[fieldId].SUMMARY.TITLE;
+
+    const expectedValue = application.EXPORTER_COMPANY[fieldId][0];
+
+    assertRow(fieldId, expectedKey, expectedValue);
+  },
+  [FINANCIAL_YEAR_END_DATE]: () => {
+    const fieldId = FINANCIAL_YEAR_END_DATE;
+    const expectedKey = FIELDS[fieldId].SUMMARY.TITLE;
+
+    const expectedValue = application.EXPORTER_COMPANY[fieldId];
+
+    assertRow(fieldId, expectedKey, expectedValue);
+  },
+  [TRADING_NAME]: () => {
+    const fieldId = TRADING_NAME;
+    const expectedKey = FIELDS[fieldId].SUMMARY.TITLE;
+
+    const expectedValue = application.EXPORTER_COMPANY[fieldId];
+
+    const expectedChangeLinkText = FIELDS[fieldId].SUMMARY.TITLE;
+
+    assertRow(fieldId, expectedKey, expectedValue, expectedChangeLinkText);
+  },
+  [TRADING_ADDRESS]: () => {
+    const fieldId = TRADING_ADDRESS;
+    const expectedKey = FIELDS[fieldId].SUMMARY.TITLE;
+
+    const expectedValue = application.EXPORTER_COMPANY[fieldId];
+
+    const expectedChangeLinkText = FIELDS[fieldId].SUMMARY.TITLE;
+
+    assertRow(fieldId, expectedKey, expectedValue, expectedChangeLinkText);
+  },
+  [WEBSITE]: () => {
+    const fieldId = WEBSITE;
+    const expectedKey = FIELDS[fieldId].SUMMARY.TITLE;
+
+    const expectedValue = application.EXPORTER_COMPANY[fieldId];
+
+    const expectedChangeLinkText = FIELDS[fieldId].SUMMARY.TITLE;
+
+    assertRow(fieldId, expectedKey, expectedValue, expectedChangeLinkText);
+  },
+  [PHONE_NUMBER]: () => {
+    const fieldId = PHONE_NUMBER;
+    const expectedKey = FIELDS[fieldId].SUMMARY.TITLE;
+
+    const expectedValue = application.EXPORTER_COMPANY[fieldId];
+
+    const expectedChangeLinkText = FIELDS[fieldId].SUMMARY.TITLE;
+
+    assertRow(fieldId, expectedKey, expectedValue, expectedChangeLinkText);
```

Comment:
Do you think we could make a helper function for each field e.g `getField`, passing the ID - which then returns they key, value and expected change link text - then pass thesee to `assertRow`? Would reduce the repetition for each row.

Also are we testing the change link HREF?